ds/data/processed/eurostat/epo_patent_applications.nuts2.csv

```diff
@@ -0,0 +1,736 @@
+year,nuts_id,nuts_year_spec,epo_patent_applications
+2016,UKC1,2010,30
```

Comment:
shouldn't these be classified as NUTS 2016?

(same of other years like 2009 -> 2006 etc)